agents/Code-Conductor.agent.md

```
@@ -320,7 +320,28 @@ Bundle-specific frame-spine semantics are deferred to #515; #512 v1 spine behavi
+     "@
+     pwsh -NoProfile -NonInteractive `
+          -File '.github/scripts/emit-pipeline-metrics-v4.ps1' `
+          -BodyFile $bodyFile `
+          -V3BaseYaml $v3Base `
+          -RichBody $richBody `
+          -IssueNumber {ISSUE_NUMBER}
+     if ($LASTEXITCODE -ne 0) {
+         Write-Warning "emit-pipeline-metrics-v4.ps1 failed (exit $LASTEXITCODE) — PR body may lack the v4 block."
+         # Non-fatal: gh pr create proceeds with whatever body is in $bodyFile (M9)
+     }
+     ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)